test/systems/plonk/placeholder/placeholder.cpp

```diff
@@ -1176,3 +1184,84 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(prover_test, F, TestFixtures) {
 }
 BOOST_AUTO_TEST_SUITE_END()
 
+
+BOOST_AUTO_TEST_SUITE(placeholder_circuit2_mnt6)
+    /* This is stub for MNT6 curve, until proper marshaling is implemented */
+    using curve_type = algebra::curves::bls12<381>;
+//    using curve_type = algebra::curves::mnt6_298;
+    using field_type = typename curve_type::scalar_field_type;
+
+    struct placeholder_test_params {
+        using merkle_hash_type = hashes::keccak_1600<256>;
+        using transcript_hash_type = hashes::keccak_1600<256>;
+
+        constexpr static const std::size_t witness_columns = 3;
+        constexpr static const std::size_t public_input_columns = 1;
+        constexpr static const std::size_t constant_columns = 0;
+        constexpr static const std::size_t selector_columns = 2;
+
+        using arithmetization_params =
+            plonk_arithmetization_params<witness_columns, public_input_columns, constant_columns, selector_columns>;
+
+        constexpr static const std::size_t lambda = 1;
+        constexpr static const std::size_t m = 2;
+    };
+    using circuit_t_params = placeholder_circuit_params<
+        field_type,
+        typename placeholder_test_params::arithmetization_params
+    >;
+
+    using transcript_type = typename transcript::fiat_shamir_heuristic_sequential<typename placeholder_test_params::transcript_hash_type>;
+
+    using commitment_scheme_params_type = nil::crypto3::zk::commitments::commitment_scheme_params_type<field_type, std::vector<std::uint8_t>>;
+    using commitment_scheme_dummy_type = dummy_commitment_scheme_type<commitment_scheme_params_type, typename placeholder_test_params::transcript_hash_type>;
+    using placeholder_params_type = nil::crypto3::zk::snark::placeholder_params<circuit_t_params, commitment_scheme_dummy_type>;
+    using policy_type = zk::snark::detail::placeholder_policy<field_type, placeholder_params_type>;
+
+    using kzg_type = commitments::batched_kzg<curve_type, typename placeholder_test_params::transcript_hash_type>;
+    using kzg_scheme_type = typename commitments::kzg_commitment_scheme<kzg_type>;
+    using kzg_placeholder_params_type = nil::crypto3::zk::snark::placeholder_params<circuit_t_params, kzg_scheme_type>;
+
+BOOST_FIXTURE_TEST_CASE(prover_test, test_initializer){
```

Comment:
Can we create something similar to placeholder_test_fixture, or re-use it? Maybe create something like placeholder_kzg_test_fixture and add it into 

using TestFixtures = boost::mpl::list<
    placeholder_test_fixture<algebra::curves::pallas, poseidon_type, poseidon_type, witness_columns_1, public_columns_1, constant_columns_1, selector_columns_1, rows_amount_1, 4>,
    placeholder_test_fixture<algebra::curves::pallas, hashes::keccak_1600<512>, hashes::keccak_1600<512>, witness_columns_1, public_columns_1, constant_columns_1, selector_columns_1, rows_amount_1, 4>
    >;